DataHub/Integration/07_ManageVariantSets.md

```diff
@@ -0,0 +1,209 @@
+
+[comment]: <> (zu prüfen)
+
+
+## Create a variant set
+
+Create a variant set to be able to create variants to products or offers. A variant set is always assigned to a certain attribute set. When a variant set is created to an attribute set, you can create variants to all products or offers with this attribute set. You must assign at least one defining attribute to a variant set. Moreover, you can assign further defining and changeable attributes.   
+
+### Define the variant set basic data
+
+Define the basic inputs, such as name and attribute set, for the variant set.
+
+#### Prerequisites
+
+An attribute set has been created, see [Create an attribute set](./02_ManageAttributeSets.md#create-an-attribute-set).
+
+> [Info] By default, the *PIM Basic Set* attribute set has been created when installing the *PIM* module.
+
+#### Procedure
+
+*PIM > Settings > Tab VARIANT SETS*  
+*Omni-Channel > Settings > Tab VARIANT SETS*
+
+![Variant sets](../../Assets/Screenshots/DataHub/Others/VariantSets.png "[Variant sets]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The *Create variant set* view is displayed.
+
+    ![Create variant set](../../Assets/Screenshots/DataHub/Others/CreateVariantSet.png "[Create variant set]")
+
+2. Enter a name for the variant set in the *Name* field.
+
+3. Select an attribute set in the *Attribute set* drop-down list. All active attribute sets are displayed in the list. Bear in mind that only those attribute sets that are relevant for the module are displayed.   
+    The *Formulas* section with the *Formula for SKU* field is displayed.
+
+    > [Info] You need to specify defining and changeable attributes for the variant set at a later step. Be aware that only those attributes assigned to the selected attribute set will be available.
+
+    ![Formulas](../../Assets/Screenshots/DataHub/Others/Formulas.png "[Formulas]")
+
+4. Enter a formula for the variants SKU in the *Formula for SKU* field using the displayed placeholders. By default, you can enter the formula **{master}-{L}**.   
+
+    > [Info] The value of the defining attributes can be included into the SKU formula. When you have added a defining attribute, the corresponding placeholder for this attribute is displayed in the *Formulas* section.
+
+5. If desired, enable the *Copy values of changeable attributes to variant when creating a new variant* toggle. It is recommended to enable it.
+
+    > [Info] This feature makes it easier for you to associate independently created products and create a master-variant relationship between them. For example, if you have the same T-shirt in different colors, all the color T-shirts should be added as variants to a master T-shirt. If this toggle is enabled, the changeable attributes and its values, which are different pro variant, are copied once when creating the variant, so that you just have to update the values if necessary. 
```

Comment:
```suggestion
    > [Info] This feature makes it easier for you to associate independently created products and create a master-variant relationship between them. For example, if you have the same T-shirt in different colors, all the colored T-shirts should be added as variants to a master T-shirt. If this toggle is enabled, the changeable attributes and its values, which are different pro variant, are copied once when creating the variant, so that you just have to update the values if necessary. 
```

Reply:
Was passiert, wenn ich den Toggle disable?